content/en/tracing/serverless_functions/enable_aws_xray.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Enable AWS X-Ray Tracing
+kind: documentation
+description: 'Trace your Lambda functions with AWS X-Ray'
+---
+## Enable AWS X-Ray
+
+**Prerequisite:** [Install the AWS integration][1].
+
+1. Ensure the following permissions are present in the policy document for your AWS/Datadog Role:
+
+```text
+xray:BatchGetTraces,
+xray:GetTraceSummaries
+```
+
+    The `BatchGetTraces` permission is used to return the full traces. The `GetTraceSummaries` permission is used to get a list of summaries of recent traces.
+
+2. [Enable the X-Ray integration within Datadog][2].
+
+3. If you are using a Customer Master Key to encrypt traces, add the `kms:Decrypt` method to your policy where the Resource is the Customer Master Key used for X-Ray.
+
+**Note:** Enabling the AWS X-Ray integration increases the amount of consumed Analyzed Spans which can increase your bill.
+
+### Enabling AWS X-Ray for your functions
+
+To get the most out of the AWS X-Ray integration:
+ - Enable it on your Lambda functions and API Gateways, either using the Serverless Framework plugin or manually; and
+ - Install the tracing libraries in your Lambda functions.
```

Comment:
Not sure why, but the bullet list isn't working here in the output. Maybe missing an empty line between 27 and 28?